controllers/activemqartemis_controller.go

```diff
@@ -311,41 +319,6 @@ func validatePodDisruption(customResource *brokerv1beta1.ActiveMQArtemis) *metav
 	return nil
 }
 
-func validateBrokerVersion(customResource *brokerv1beta1.ActiveMQArtemis) *metav1.Condition {
```

Comment:
`validateBrokerVersion` seems only used in `controllers/activemqartemis_controller.go`, why was it moved?